nautobot/core/templates/generic/object_retrieve.html

```diff
@@ -86,6 +108,13 @@ <h1>
                 Advanced
             </a>
         </li>
+        {% if associated_contacts_table %}
+            <li role="presentation"{% if request.GET.tab == 'contacts' %} class="active"{% endif %}>
+                <a href="{{ object.get_absolute_url }}#contacts" onclick="switch_tab(this.href)" aria-controls="contacts" role="tab" data-toggle="tab">
+                    Contacts/Teams
```

Comment:
Maybe not for this PR, but John and I were bouncing around the idea of using mdi icons for some of the detail view tabs instead of text so as to mitigate tab sprawl on views like Device detail, i.e. instead of: 

![image](https://github.com/nautobot/nautobot/assets/5603551/e7a9cb18-a461-4728-804f-85e7effb9d7d)

something like:

![image](https://github.com/nautobot/nautobot/assets/5603551/aa531bd2-9402-45c9-b0e2-7fab55da83d7)